decidim-design/app/helpers/decidim/design/announcement_helper.rb

```diff
@@ -37,15 +37,15 @@ def announcement_sections
             ]
           },
           {
-            id: "plain_text_vs_hash",
+            id: t("decidim.design.helpers.plain_text"),
             contents: [
               {
                 type: :text,
-                values: ["You can provide as first argument both a plain text and a hash object"]
+                values: [t("decidim.design.helpers.plain_text_description")]
               },
               {
                 type: :table,
-                options: { headings: ["Announcement", "Callout class", "Argument"] },
+                options: { headings: [t("decidim.design.components.announcement.title"), t("decidim.design.helpers.callout_class"), t("decidim.design.helpers.argument")] },
                 items: announcement_table(
                   { text: { title: "This is the title", body: "This is the body" }, callout_class: "success",
```

Comment:
I would add I18n support for this text, "This is the title" and "This is the body" starting line 50 to 71